README.md

```diff
@@ -44,35 +43,31 @@ cd nhs-england-tools/repository-template
 
 ### Prerequisites
 
-The following software packages or their equivalents are expected to be installed
-
-- [GNU make](https://www.gnu.org/software/make/) **v3.82 or later: OS X users, take note below**
-- [Docker](https://www.docker.com/)
+The following software packages, or their equivalents, are expected to be installed:
 
-Note that the version of GNU Make available by default on OS X is earlier than this.  You will need to upgrade it, or certain `make` tasks will fail.  On OS X, you will need [homebrew](https://brew.sh/) installed, then to install `make`, like so:
-
-```shell
-/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-brew install make
-```
+- [docker](https://www.docker.com/) container runtime or a compatible tool, e.g. [podman](https://podman.io/),
+- [asdf](https://asdf-vm.com/) version manager,
+- [GNU make](https://www.gnu.org/software/make/) 3.82 or later,
+- [GNU coreutils](https://www.gnu.org/software/coreutils/) and [GNU binutils](https://www.gnu.org/software/binutils/) may be required to build dependencies like Python, which may need to be compiled during installation. For macOS users, this has been scripted and automated by the `dotfiles` project; please see this [script](https://github.com/nhs-england-tools/dotfiles/blob/main/assets/20-install-base-packages.macos.sh) for details.
 
-You will then see instructions to fix your `$PATH` variable to make the newly installed version available.
-If you are using [dotfiles](https://github.com/nhs-england-tools/dotfiles), this is all done for you.
+> [!NOTE]<br>
+> The version of GNU make available by default on macOS is earlier than 3.82. You will need to upgrade it or certain `make` tasks will fail. On macOS, you will need [homebrew](https://brew.sh/) installed, then to install `make`, like so:
+>
+> ```shell
+> /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
If we've told them to install homebrew above, I don't think we need this line here.  The way it reads is "install brew, then install brew again".  I know that's what I wrote originally, but reading it back now it feels a bit odd.

Reply:
Thanks. I've removed that line from the `README.md` file